### ISSUE TITLE
How do you call com.nhaarman.mockito_kotlin.mock from Java?

### ISSUE THREAD

**Issue by jplipata** [nhaarman/mockito-kotlin#306]
I am able to use it just fine in a Kotlin class.  

However, when I try to use `mock()` in a java class, mockito_kotlin's `mock()` or `mock<T>()` does not resolve.

I'm using `androidTestImplementation "com.nhaarman:mockito-kotlin:1.6.0"` in the `androidTest` environment.

I've tried manually importing:
import com.nhaarman.mockito_kotlin.mock.*;
import com.nhaarman.mockito_kotlin.mock.MockitoKt.*;
etc.
but the only `mock()` method that resolves is `import static org.mockito.Mockito.mock;`

TLDR: I'm able to get mockito_kotlin's `mock()` to work in Kotlin classes, but in Java it's not resolving for me

**Comment by pdegand** [comment]
You can't.

`mock<T>()` is an inline fun with a reifeid generic and those cannot be used in Java.

**Comment by jplipata** [comment]
That explains it! Thanks @pdegand
